src/fellows.ts

```diff
@@ -1,8 +1,23 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
 /* eslint-disable @typescript-eslint/ban-ts-comment */
-import { ApiPromise, WsProvider } from "@polkadot/api";
+import { createClient } from "@polkadot-api/client";
+import { getChain } from "@polkadot-api/node-polkadot-provider";
+import { getSmProvider } from "@polkadot-api/sm-provider";
+import { WebSocketProvider } from "@polkadot-api/ws-provider/node";
+import { start } from "smoldot";
 
+import collectiveDescriptor from "./codegen/collectives";
+import relayDescriptor from "./codegen/relay";
 import { ActionLogger } from "./github/types";
 
+const collectiveChain =
+  require("@substrate/connect-known-chains/polkadot_collectives") as {
+    chainSpec: string;
+  };
+const polkadot = require("@substrate/connect-known-chains/polkadot") as {
+  chainSpec: string;
+};
```

Comment:
I'm having a problem with this imports. 

I can not convert it to a regular import as it gives me the error `SyntaxError: Cannot use import statement outside a module

@kratico do you have an idea of how we could fix this problem?

I have been scratching my head for too long